src/configuration/Configuration.h

```diff
@@ -46,20 +48,28 @@ namespace SlimeVR {
             CalibrationConfig getCalibration(size_t sensorID) const;
             void setCalibration(size_t sensorID, const CalibrationConfig& config);
 
+            size_t getWiFiCredentialCount() const;
+            const std::map<const char*, WiFiCredential>* const getWiFiCredentials() const;
+            void setWiFiCredential(const WiFiCredential& config);
+            void removeWiFiCredential(const char* ssid);
```

Comment:
opinion: we should start using c++17 `std::string_view`

Reply:
suggestion: fix up the esp32 toolchain 🙃